### PR TITLE
Add missing commonLabels

### PR DESCRIPTION
**Pull request by janwar73**
…l in version 0.19.0

### CODE REVIEW

**Comment by janwar73** [comment]
Send CLA to cla@trino.io for approval

**Comment by martint** [comment]
@cla-bot check

File: charts/trino/templates/service.yaml [hunk 1]
```diff
@@ -7,6 +7,9 @@ metadata:
     chart: {{ template "trino.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    {{- if .Values.commonLabels }}
```
Comment (nineinchnick) [inline charts/trino/templates/service.yaml:10]: Can you also add them to `secret.yaml`?

**Review by nineinchnick — APPROVED** [review]

**Comment by janwar73** [comment]
Who can merge this. I don’t see the merge option

**Review by wendigo — APPROVED** [review]

**Comment by wendigo** [comment]
@janwar73 Can you fix both commit messages: we require first line to be shorter than 80 chars to render nicely in the GH

**Comment by janwar73** [comment]
Done, thanks

File: charts/trino/templates/service.yaml [hunk 2]
```diff
@@ -24,4 +27,4 @@ spec:
   selector:
     app: {{ template "trino.name" . }}
     release: {{ .Release.Name }}
-    component: coordinator
+    component: coordinator
```
Comment (nineinchnick) [inline charts/trino/templates/service.yaml:30]: Revert this - configure your text editor not to remove end of line at the end of file.

Comment (bond-) [reply charts/trino/templates/service.yaml]: Done

**Comment by janwar73** [comment]
Who merges it now

**Comment by mosabua** [comment]
Looks good. Thank you @janwar73 . We will cut a new release soon as well hopefully.

**Comment by janwar73** [comment]
Thank you all,

**Comment by bond-** [comment]
@mosabua How may I request a new release with this fix?

**Pull request merged**